src/Castle.Core/DynamicProxy/CustomAttributeInfo.cs

```diff
@@ -34,26 +36,26 @@ public class CustomAttributeInfo : IEquatable<CustomAttributeInfo>
 		// Cached empty arrays to avoid unnecessary allocations
 		private static readonly PropertyInfo[] EmptyProperties = new PropertyInfo[0];
 		private static readonly FieldInfo[] EmptyFields = new FieldInfo[0];
-		private static readonly object[] EmptyValues = new object[0];
+		private static readonly object?[] EmptyValues = new object?[0];
```

Comment:
Maybe this PR would be a good opportunity to change such declarations to simply use `var` instead?